content/source/docs/enterprise/before-installing/index.html.md

```diff
@@ -126,7 +126,7 @@ Some operating systems have specific configuration requirements:
 
 For other Linux distributions, check Docker compatibility:
 
-* The instance should run a supported version of Docker engine (1.7.1 or later, minimum 17.06.2-ce, maximum 18.09.2). This also requires a 64-bit distribution with a minimum Linux Kernel version of 3.10.
+* The instance should run a supported version of Docker engine (1.7.1 or later, minimum 17.06.2-ce, maximum 19.03.8). This also requires a 64-bit distribution with a minimum Linux Kernel version of 3.10.
     * Replicated 2.32.0 and above required when running Docker 18+.
```

Comment:
We already call out replicated versions here.

Reply:
This call out didn't age well and is misleading in its current form. Reading this, you'd think Replicated 2.32.2 is all that's necessary to run Docker 18+, including Docker 19+. That is not the case. This is why I don't recommend notes like this for the installation guide. Something like "Support for Docker 18.y.z was added in Replicated 2.y.z" would have been better.